### ISSUE TITLE
connect popup selector not working with keyboard (wcag-aa issue)

### ISSUE THREAD

**Issue by userquin** [npmx-dev/npmx.dev#553]
Once opened we should be able to use arrow up/down to traverse the options, right now we need to use tab to traverse them. 

For example, package manager selector is fine (we should close the popup when focus lost):

<img width="1015" height="446" alt="package manager selector" src="https://github.com/user-attachments/assets/10d12142-fddf-4dbd-b6c9-5d076e8b8b5c" />

**Comment by nulfrost** [comment]
I was looking for which pattern would be most suitable for this and found this: https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-select-only/

I also went to check out the React equivalent in some component libraries and they match the behaviour in the APG
- [Ariakit Select](https://ariakit.org/components/select)
- [React Aria Select](https://react-aria.adobe.com/Select)

Probably a good place to start, if no one else picks this up I can try some stuff out

**Comment by DDeenis** [comment]
Is this still a problem? I just tried it out, and arrows navigation works fine
Windows 10, latest Chrome

**Comment by nulfrost** [comment]
oh huh, yeah looks like this was addressed

**Comment by RYGRIT** [comment]
This issue can be closed; I tested it at https://main.npmx.dev, and the problem no longer exists. @serhalp @userquin
